<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Sonde</class>
 <widget class="QWidget" name="F_Sonde">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>310</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>300</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Acquisition</string>
  </property>
  <widget class="QLabel" name="Lb_Valeur">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">position:fixed;</string>
   </property>
   <property name="text">
    <string>Valeur :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_Enregistrement">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Lcd_Valeur">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>28</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">position:fixed;
background-color:white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_Lancer">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="TbW_Valeurs">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>9</y>
     <width>156</width>
     <height>251</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>37</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="Lb_DateHeure">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>212</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">position:fixed;</string>
   </property>
   <property name="text">
    <string>Date et heure du début :</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_Intervalle">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">position:fixed;</string>
   </property>
   <property name="text">
    <string>Durée entre 2 mesures (secondes):</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTimeEdit" name="TiE_HeureAcquisition">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>278</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">position:fixed;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_ModeAcquisition">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>68</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">position:fixed;</string>
   </property>
   <property name="text">
    <string>Automatique</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_TpsAcquisition">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>156</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">position:fixed;</string>
   </property>
   <property name="text">
    <string>Temps d'acquisition (minutes) :</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="SBx_TempsAcquisition">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>190</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">position:fixed;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SBx_Intervalle">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>128</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">position:fixed;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="DtE_DateAcquisition">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>250</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">position:fixed;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
